ccd-definition/ComplexTypes/CareSupervision/AdditionalApplicationsBundle.json

```diff
@@ -8,6 +8,249 @@
     "SecurityClassification": "Public",
     "Searchable": "N"
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "AdditionalApplicationsBundle",
+    "ListElementCode": "c2DocumentBundleConfidential",
+    "FieldType": "C2DocumentBundle",
```

Comment:
Looks like we need a `C2DocumentBundleConfidential` type and its  `CategoryID` is `"c2ApplicationsConfidential"`.

And I guess we need a Confidential subfolder under C2 application category in CFV